radiation/python/radlw/rtrnmc_gt4py.py

```diff
@@ -0,0 +1,584 @@
+                                                      backend,
+                                                      shape_nlp1,
+                                                      type_nbands)
+    elif var == 'fracs' or var == 'tautot' or var == 'cldfmc':
+        indict_gt4py[var] = create_storage_from_array(indict[var],
+                                                      backend,
+                                                      shape_nlp1,
+                                                      type_ngptlw)
+    elif var == 'pklay' or var == 'pklev':
+        indict_gt4py[var] = create_storage_from_array(indict[var],
+                                                      backend,
+                                                      shape_nlp1,
+                                                      type_nbands)
+    else:
+        indict_gt4py[var] = indict[var]
+
+locvars = ['clrurad', 'clrdrad', 'toturad', 'totdrad', 'gassrcu', 'totsrcu',
+           'trngas', 'efclrfr', 'rfdelp', 'fnet', 'fnetc', 'totsrcd',
+           'gassrcd', 'tblind', 'odepth',
+           'odtot', 'odcld', 'atrtot', 'atrgas', 'reflct', 'totfac',
+           'gasfac', 'flxfac', 'plfrac', 'blay', 'bbdgas', 'bbdtot',
+           'bbugas', 'bbutot', 'dplnku', 'dplnkd', 'radtotu', 'radclru',
+           'radtotd', 'radclrd', 'rad0', 'clfm', 'trng', 'gasu', 'itgas',
+           'ittot', 'ib']
+
+locdict_gt4py = dict()
+
+for var in locvars:
+    if var[-3:] == 'rad':
+        locdict_gt4py[var] = create_storage_zeros(backend,
+                                                  shape_nlp1,
+                                                  type_nbands)
+    elif var == 'fnet' or var == 'fnetc' or var == 'rfdelp':
+        locdict_gt4py[var] = create_storage_zeros(backend,
+                                                  shape_nlp1,
+                                                  type1)
+    elif var == 'itgas' or var == 'ittot':
+        locdict_gt4py[var] = create_storage_zeros(backend,
+                                                  shape_nlp1,
+                                                  (np.int32, (ngptlw,)))
+    elif var == 'ib':
+        locdict_gt4py[var] = create_storage_zeros(backend,
+                                                  shape_2D,
+                                                  np.int32)
+    else:
+        locdict_gt4py[var] = create_storage_zeros(backend,
+                                                  shape_nlp1,
+                                                  type_ngptlw)
+
+ngb = np.array(ngb)
+NGB = np.tile(ngb[None, None, :], (npts, 1, 1))
+NGB = create_storage_from_array(NGB,
+                                backend,
+                                shape_2D,
+                                (np.int32, (140,)),
+                                default_origin=(0, 0))
+
+rec_6 = 0.166667
+tblint = ntbl
+flxfac = wtdiff * fluxfac
+lhlw0 = True
+
+@stencil(backend, rebuild=False, externals={'rec_6': rec_6,
```

Comment:
you might want to consider using a global rebuild flag, but while developing it can make sense to set specific stencils

Reply:
Woops thought I had done that. Now changed to use a global flag